src/compiler-host.js

```diff
@@ -239,7 +239,11 @@ export default class CompilerHost {
     if (!compiler) {
       compiler = this.fallbackCompiler;
 
-      let { code, binaryData, mimeType } = await compiler.get(filePath);
+      if (!compiler) {
+        compiler = this.getPassthroughCompiler();
```

Comment:
I'm not sure that this is correct, for complicated weird reasons. Do you have a repro file that you hit this with?